### PR TITLE
Penguin seal page

### PR DESCRIPTION
**Pull request by dnbute**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after), along with a short summary of changes:

## Issue

Fixes #<gh-issue-id>

## Changelog:

- Added `align-bottom` mixin that justifies the text and aligns it at the bottom of the section
- Made `narrow` section with `bg-image` behave as expected
- Modified LCP logic to check background images as well

## Test URLs:
- Original: https://www.sunstar.com/healthy-thinking/penguin-indoor-seal
- Before: https://main--sunstar--hlxsites.hlx.live/healthy-thinking/penguin-indoor-seal
- After: https://penguin-seal--sunstar--hlxsites.hlx.live/healthy-thinking/penguin-indoor-seal

## Library

- [ ] New Blocks introduced in this PR
      If yes, please provide details below
**Block Name** : For e.g. _cards_
- [ ] Documented in sidekick library

- [ ] New variations introduced in this PR
**Variation Name** :  For e.g. _cards (grid)_
- [ ] Documented in sidekick library

- [x] New mixins introduced in this PR
**Mixin name**: `align-bottom` 
- [x] Documented in sidekick library

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -1005,6 +1018,13 @@ main .block.download a::after {
     max-width: 54.5rem;
   }
 
+  main .section.narrow.section.with-background-image > .section-container {
```
[the remark this answers is not in the document]
Comment (dnbute) [reply styles/styles.css:1021]: Now that you mention it, no, the problem was that the `position:relative` didn't work with the background images, I'll look for more cases like this 👍

[the remark this answers is not in the document]
Comment (dnbute) [reply styles/styles.css]: Seems only `narrow` and `narrower` needed this compatibility fix

File: scripts/scripts.js [hunk 2]
```diff
@@ -25,10 +25,11 @@ const LCP_BLOCKS = [
   'overlapping-content',
   'carousel',
   'career-hero',
+  'background-image',
 ]; // add your LCP blocks to the list
 const SKIP_FROM_LCP = ['breadcrumb']; // add blocks that shouldn't ever be LCP candidates to the list
 // search for at least these many blocks (post-skipping-non-candidates) to find LCP candidates
-const MAX_LCP_CANDIDATE_BLOCKS = 2;
+const MAX_LCP_CANDIDATE_BLOCKS = 3;
```
[the remark this answers is not in the document]
Comment (dnbute) [reply scripts/scripts.js]: I've reverted this and made some changes, wdyt

File: styles/styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (sdmcraft) [reply styles/styles.css]: Not sure I get the reason. The section with background gets relatively positioned and the section container in it is absolutely positioned in its relation. That should happen always unless there's a scenario I may be missing.
How does the UI break if you don't add this style?

[the remark this answers is not in the document]
Comment (dnbute) [reply styles/styles.css]: Screenshot 2023-11-23 at 14 12 35

And this is what should happen, screenshot taken from the branch url

[the remark this answers is not in the document]
Comment (sdmcraft) [reply styles/styles.css]: ok I see the reason now. The `narrow` and `narrower` sections are relatively positioned while we need them absolutely positioned. I would have wanted to get rid of this relative positioning of `narrow` and `narrower` sections. It seems to be coming from the previous project for Sunstar Engg. site but not sure what impact its going to create. Let's go with what you propose for now.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**